lib/loadTransforms.js

```diff
@@ -1,17 +1,9 @@
 "use babel";
 import { transformsPath } from "./scaffold";
 
-module.exports = function loadTransforms() {
+module.exports = async function loadTransforms() {
```

Comment:
Does this impact supported atom versions? Does it pull in regenerator runtime or anything?

Reply:
I'm sure it breaks support for some old atom versions, but trying to determine which is proving difficult...it certainly doesn't pull in any new dependencies though.

Reply:
Atom has used `stage: 0` in the babel options for `'use babel';` support since at least 2015: https://github.com/atom/atom/commit/49e37f541046f0ce9037494d3fe9bf7e85847980#diff-bb35c33015964b91f5c83deb2a33a814